packtools/sps/validation/alternatives.py

```diff
@@ -4,48 +4,92 @@
-        if self.obtained_parent not in self.parent_children_dict:
-            yield self.create_validation_response(
-                expected=list(self.parent_children_dict.keys()),
-                obtained=self.obtained_parent,
-                advice=f"Provide parent tag according to the list: {list(self.parent_children_dict.keys())}"
-            )
+            Params
+            ------
+            alternative : dict, such as:
+                {
+                    'alternative_children': ['graphic', 'table'],
+                    'alternative_parent': 'table-wrap',
+                    'parent': 'article',
+                    'parent_id': None
+                }
 
-    def validate_children(self):
+            children_list : list, such as:
+                ["graphic", "table"]
+
+            Returns
+            -------
+            list[dict], such as:
+                [
+                    {
+                        'title': 'Alternatives validation',
+                        'parent': 'article',
+                        'parent_id': None,
+                        'item': 'table-wrap',
+                        'sub_item': 'alternatives',
+                        'validation_type': 'value in list',
+                        'expected_value': ['graphic', 'table'],
+                        'got_value': ['p'],
+                        'response': 'ERROR',
+                        'message': "Got ['p'], expected ['graphic', 'table']",
+                        'advice': "Provide child tags according to the list: ['graphic', 'table']",
+                        'data': {
+                            'alternative_children': ['p'],
+                            'alternative_parent': 'table-wrap',
+                            'parent': 'article',
+                            'parent_id': None
+                        }
+                    },
+                ]
+        """
         for tag in self.obtained_children:
-            if tag not in (self.parent_children_dict.get(self.obtained_parent) or []):
-                yield self.create_validation_response(
-                    expected=self.parent_children_dict.get(self.obtained_parent),
+            if tag not in (self.children_list or []):
+                yield format_response(
+                    title="Alternatives validation",
+                    parent=self.alternative.get("parent"),
+                    parent_id=self.alternative.get("parent_id"),
+                    item=self.alternative.get("alternative_parent"),
+                    sub_item="alternatives",
+                    validation_type="value in list",
+                    is_valid=False,
+                    expected=self.children_list,
                     obtained=self.obtained_children,
-                    advice=f"Provide child tags according to the list: {self.parent_children_dict.get(self.obtained_parent)}"
+                    advice=f"Provide child tags according to the list: {self.children_list}",
```

Comment:
@Rossi-Luciano no lugar de `Provide child tags according to the list: {self.children_list}`, use `Add {self.children_list} as sub-elements of {self.alternative.get("parent")}/alternatives`

Reply:
OK, @robertatakenaka